src/app/app-layout/navbar/navbar.component.ts

```diff
@@ -29,16 +39,36 @@ export class NavbarComponent {
               private router: Router) {
   }
 
+  ngOnInit() {
+    this.forceCloseDropdownOnResize()
+  }
+
+  // Handle undefined state where dropdown remains opened if screen is 
+  // resized from smaller to bigger while dropdown is open
+  forceCloseDropdownOnResize() {
+    fromEvent(window, 'resize').pipe(debounceTime(300)).subscribe(event => {
+      let breakWidth = 
+        resolveConfig(tailwindConfig)
+        .theme.screens?.lg
+        .replace('px','') ?? '0'
+      if(parseInt(breakWidth) < (event.currentTarget as Window).innerWidth) {
+        this.appLayoutStore.forceCloseDropdownMenu()
+      }
+    })
+  }
```

Comment:
I'll try to figure this out later, looks a little bit hacky...

Reply:
It's a part of state management to keep the $isDropdownMenuOpen consistent with what is on the screen.

There was one case - when dropdown is open and user resizes screen to be bigger than 1024px, the layout changes but $isDropdownOpen remained as "true". To avoid this possible "hanging" state, this code forces the dropdown to close when it exceeds 1024px.

I didn't want to use magic numbers so I imported tailwind config to reference the "large" screen breakpoint.